src/main/java/build/buildfarm/common/redis/BalancedRedisQueue.java

```diff
@@ -96,8 +100,9 @@
   public BalancedRedisQueue(String name, List<String> hashtags) {
     this.originalHashtag = RedisHashtags.existingHash(name);
     this.name = RedisHashtags.unhashedName(name);
+    this.type = "regular";
```

Comment:
this is good. we'll turn it into a protobuf enum later or something.  And this will have no performance impact since the dispatching is only on object construction.  existing users should not notice.